Mathlib/Algebra/Order/Monoid/Lemmas.lean

```diff
@@ -314,6 +314,27 @@ by simp_rw [min_le_iff, le_max_iff]; contrapose! h; exact mul_lt_mul_of_lt_of_lt
 #align min_le_max_of_add_le_add min_le_max_of_add_le_add
 #align min_le_max_of_mul_le_mul min_le_max_of_mul_le_mul
 
+end LinearOrder
+
+section LinearOrder
+variable [LinearOrder α] [CovariantClass α α (· * ·) (· ≤ ·)]
+  [CovariantClass α α (swap (· * ·)) (· ≤ ·)] {a b c d : α}
```

Comment:
I think we made a global change to use `HMul.hMul` and `LE.le` instead, for performance?

Reply:
Mario has a ongoing PR to core, so surely it doesn't matter?

Reply:
We decided it mattered enough to exhaustively use the workaround in the meantime at every other call site; so we should do the same here.